src/ODINN.jl

```diff
@@ -59,6 +57,9 @@ cd(@__DIR__)
 const global root_dir::String = dirname(Base.current_project())
 const global root_plots::String = joinpath(root_dir, "plots")
 
+const SYSIMAGE_DIR = joinpath(homedir(), ".ODINN")
+const SYSIMAGE_PATH = joinpath(SYSIMAGE_DIR, "odinn_sysimage.so")
+ENV["JULIA_DEPOT_PATH"] = joinpath(homedir(), ".julia")  # Ensure shared cache
```

Comment:
@JordiBolibar what is this needed for?

Reply:
This is just for PackageCompiler.jl to create a system image. I have so far commented that part of the code, so I'll also comment this line.